rtlib/generic_opts.hh

```diff
@@ -24,8 +24,11 @@
 #ifndef RTLIB_GENERIC_OPTS_HH_
 #define RTLIB_GENERIC_OPTS_HH_
 
-
-#include <unistd.h>
+extern "C" {
+  #include <getopt.h>
```

Comment:
The [generic_opts.hh](https://github.com/jlab/gapc/blob/checkpointing/rtlib/generic_opts.hh) file uses the standard C command line parser (defined in `getopt.h`; the include statement was missing before for some reason) instead of `boost`, so I used it as well here.

Reply:
ups, didn't recognize that it was missing. Thanks for adding!